src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,14 @@
+@import "variables";
```

Comment:
you seem to have lost few files on commit
there's no styling for `month-lenght` class